src/Utils/GeneralComponents/useCountdown.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+import moment from "moment";
+import "moment-duration-format";
```

Comment:
please change the name of the `GeneralComponents` folder that contains it to `TimingUtils`.